legacy-content-import/src/main/scala/legacycontentimport/Article.scala

```diff
@@ -19,53 +17,59 @@ case class Article(
 
 object Article {
 
-  val excludedTags = Seq("help/help", "tone/help")
+  private val excludedTags = Seq("help/help", "tone/help")
 
-  def fromCapiHelpSection(capiDomain: String, capiKey: String): Seq[Article] = {
-
-    val fetch = fromCapiHelpSectionPage(capiDomain, capiKey) _
+  private case class ArticlesAndPageCount(articles: Seq[Article], pageCount: Int)
 
-    @tailrec
-    def go(pageIndex: Int, acc: Seq[Article]): Seq[Article] =
-      Try(fetch(pageIndex)) match {
-        case Failure(_)        => acc
-        case Success(articles) => go(pageIndex + 1, acc ++ articles)
-      }
+  def fromCapiHelpSection(capiDomain: String, capiKey: String): Seq[Article] = {
+    val fetch = pageFromCapiHelpSection(capiDomain, capiKey) _
+    val articlesAndPageCount = firstPageFromCapiHelpSection(capiDomain, capiKey)
+    val pageIndices = 1 until articlesAndPageCount.pageCount
```

Comment:
Are you sure you need `until` and not `to`? Just double-checking for one-off errors.

Reply:
Yes.  Index 0 is the first page.  Then this range goes from 1 to the value of `pages`, exclusive.
When it's included as a parameter in the Capi query I add 1 to it because the Capi page indices are actually 1-based.
So if `pages` = 5, it will fetch page 1 and then 2 to 5.